cpp/src/common/row_record.h

```diff
@@ -86,9 +86,31 @@ struct Field {
         }
     }
 
+    template <typename T>
+    FORCE_INLINE T get_value() {
+        switch (type_) {
+            case common::TSDataType::BOOLEAN:
+                return value_.bval_;
+            case common::TSDataType::INT32:
+                return value_.ival_;
+            case common::TSDataType::INT64:
+                return value_.lval_;
+            case common::TSDataType::FLOAT:
+                return value_.fval_;
+            case common::TSDataType::DOUBLE:
+                return value_.dval_;
+            // case common::TSDataType::TEXT :
+            //     return value_.sval_;
```

Comment:
Why do you comment TEXT?

Reply:
TEXT is not supported in current C++ versions.It's also commented in the line 79-82.